.github/workflows/build-petclinic-overhead-base.yml

```diff
@@ -0,0 +1,25 @@
+name: publish-petclinic-base-image
+on: workflow_dispatch
```

Comment:
How about adding a path trigger?

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/.github/workflows/build-smoke-dist-fakebackend.yml#L4

Reply:
I love it!  Thanks for the idea!